react/cra-template/template/azp-templates/publish.yml

```diff
@@ -10,6 +10,11 @@ steps:
   - publish: "${{ parameters.package_directory }}/cypress"
     displayName: "Publish Test Suite"
     artifact: "${{ parameters.pipeline_cypress_artifact_name }}"
+  - task: PublishPipelineArtifact@1
```

Comment:
copying the env.sh script as a pipeline attribute to make it available to the deploy stages